src/components/OrderBook/LevelList.tsx

```diff
@@ -141,16 +142,23 @@ class LevelList extends React.Component<LevelListProps> {
     );
 
     levels.forEach((l, i: number) => {
```

Comment:
I think that this code should be slightly refactored. `level`, `index` are better.